notebooks/book2/02/sub_super_gauss_plot.ipynb

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
<p>Have you understood the use of <code>width_scale_factor</code> here?</p><p>If not I would recommend you to look at code of <code>latexify</code> : https://github.com/probml/probml-utils/blob/main/probml_utils/plotting.py#:~:text=def%20latexify(,)%3A</p><p> </p><p>In short,</p><p>Our Paper width is 6 inches (in)</p><p>when you give   <code>width_scale_factor=1.70</code> it will generate figures having width 6/1.7 = 3.53 in</p><p>Now we are putting two figures in a row,  so it will take 3.53 + 3.53 = 7.1in which is greater than our paper width 6 in and we don't want to compress our figures to make total width = 6. </p><p> </p><p>So now what <code>width_scale_factor</code> should you choose? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Okay, I am figuring it out and will share the image